library_generation/utils/utilities.py

```diff
@@ -130,10 +130,6 @@ def prepare_repo(
         # use absolute path because docker requires absolute path
         # in volume name.
         absolute_library_path = str(Path(library_path).resolve())
-        if absolute_library_path in libraries:
```

Comment:
We don't need this check because the Generation object is validated when reaches this point.